pkg/component/etcd/role/values.go

```diff
@@ -27,8 +27,8 @@ type Values struct {
 	Namespace string
 	// Rules holds all the PolicyRules for this Role
 	Rules []rbacv1.PolicyRule
-	// OwnerReferences are the OwnerReferences of the Role.
-	OwnerReferences []metav1.OwnerReference
+	// OwnerReference are the OwnerReference of the Role.
+	OwnerReference *metav1.OwnerReference
```

Comment:
did not understand what is the need to have a pointer to OwnerReference. Can it be ever nil? Same question for all occurrences

Reply:
Fixed - https://github.com/gardener/etcd-druid/pull/559/commits/abf318ef96412efa2f2d98f052ed80a8221aee75